### PR TITLE
CB-10337: Changing whitelist guide to remove confusion about iOS

### PR DESCRIPTION
**Pull request by rakatyal**

### CODE REVIEW

**Comment by riknoll** [comment]
LGTM

File: www/docs/en/dev/guide/appdev/whitelist/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ accessed.  By default, new apps are configured to allow access to any site.
 Before moving your application to production, you should formulate a whitelist
 and allow access to specific network domains and subdomains.
 
-For Android and iOS (as of their 4.0 releases), Cordova's security policy is extensible via a plugin
+For Android (as of it's 4.0 releases), Cordova's security policy is extensible via a plugin
```
Comment (riknoll) [inline www/docs/en/dev/guide/appdev/whitelist/index.md]: its not it's

Comment (riknoll) [inline www/docs/en/dev/guide/appdev/whitelist/index.md]: and release not releases

Comment (rakatyal) [inline www/docs/en/dev/guide/appdev/whitelist/index.md]: Corrected.

**Comment by sarangan12** [comment]
LGTM

**Pull request closed without merging**